mzLib/MassSpectrometry/MzSpectra/SpectralSimilarity.cs

```diff
@@ -302,6 +302,23 @@ private double[] NormalizeSpectrumSum(double[] spectrum)
             return sum;
         }
 
+        //This formula created by Brian Searle and reported at ASMS 2022 in poster "Scribe: next generation library searching for DDA experiments"
+        public double? SearleSimilarity()
+        {
+            double squaredSumDifferences = 0;
+
+            //there must be some legitimate pairs to enter this function so no need to test if pairs exist
+            foreach ((double, double) pair in _intensityPairs)
+            {
+                squaredSumDifferences += Math.Pow((pair.Item1 - pair.Item2),2);
+            }
+            if(squaredSumDifferences > 0)
+            {
+                return Math.Log(Math.Pow(squaredSumDifferences,-1));   
```

Comment:
Could use ternary operator for this type of return:
`return squaredSumDifferences > 0 ? Math.Log(Math.Pow(squaredSumDifferences,-1)) : double.MaxValue;`

But looks great!

Reply:
This is a natural logarithm. Is that the intended one?

Reply:
> This is a natural logarithm. Is that the intended one?

yep

Reply:
> return squaredSumDifferences > 0 ? Math.Log(Math.Pow(squaredSumDifferences,-1)) : double.MaxValue;

good idea. my brain is not on this level. made the switch just so that I can try to remember that such a thing even exists. thanks.